source/js/src/navigation.js.es6

```diff
@@ -1,3 +1,31 @@
+function handleKeyboardAction ({ event, onOpen, onEscape }) {
```

Comment:
why?

Reply:
Just reduces a ton of copy and paste—we use 80% of this function in 5 places.

It is basically for accessibility:
 - I click `space` or `enter` and I want to simulate a "click" and do that something.
 - I click `escape` and I want to "exit".

Reply:
Will definitely request a review on the JS side!